libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -1030,6 +1031,13 @@ void AP_AHRS_DCM::set_home(const Location &loc)
     _home = loc;
     _home.options = 0;
     _home_is_set = true;
+
+    // log ahrs home and ekf origin dataflash
+    AP::ahrs().Log_Write_Home_And_Origin();
```

Comment:
What's up with the `AP::ahrs()`? This is inside the AHRS, you just need to call the method...